app/services/data/get-reduction-by-id.js

```diff
@@ -0,0 +1,15 @@
+const config = require('../../../knexfile').web
+const knex = require('knex')(config)
+
+module.exports = function (id) {
+  var whereObject = {}
+  if (id !== undefined) {
+    whereObject.id = id
+  }
+
+  return knex('reductions')
+    .where(whereObject)
+    .then(function (reduction) {
+      return reduction[0]
```

Comment:
Why is this returning the first element?

Reply:
I want to return an object rather than an array of objects here

Reply:
Update to include fields in the select statement (I realised having no select statement is doing a `select *` and I hate `select *`)

Reply:
select * on a table is not too bad. using * on a command prompt is dangerous (rm *) :-)